dev-portal/public/index.html

```diff
@@ -17,34 +17,10 @@
     <script type="text/javascript" src="%PUBLIC_URL%/apigateway-js-sdk/lib/apiGatewayCore/simpleHttpClient.js"></script>
     <script type="text/javascript" src="%PUBLIC_URL%/apigateway-js-sdk/lib/apiGatewayCore/utils.js"></script>
     <script type="text/javascript" src="%PUBLIC_URL%/apigateway-js-sdk/apigClient.js"></script>
-
-    <script src="%PUBLIC_URL%/jquery-2.1.4.min.js"></script>
-    <script type="text/javascript">
-    // browser-detection removed from jQuery
-    // this resolves a jquery.ba-bbq error
-    jQuery.browser = {};
-    (function () {
-        jQuery.browser.msie = false;
-        jQuery.browser.version = 0;
-        if (navigator.userAgent.match(/MSIE ([0-9]+)\./)) {
-            jQuery.browser.msie = true;
-            jQuery.browser.version = RegExp.$1;
-        }
-    })();
 </script>
```

Comment:
I think this is a leftover closing script tag? Doesn't appear to be matched with anything.

Reply:
Good call.